tests/serialization/test_deserialization.py

```diff
@@ -0,0 +1,71 @@
+"""Deserialization tests."""
```

Comment:
These are just deserialization tests for Acquisition Functions, I think the module description should thus be a bit more descriptive and clear.

Reply:
Not quite. This module is not about testing acquisition functions but to test the core deserialization that might occur. It's simply the case that we have `abbreviation` only for ACQFs, hence I use those for the tests. But the focus here is to check if deserialization works for all kinds of provided (and missing) type information

Reply:
Ok, because I would interpret this as "This is the place where we test all of the deserialisation" which we do not do (roundtrips also test that in some sense), but I get your point. Still, I think a slightly more informative docstring would be beneficial, but if you disagree, feel free to resolve.

Reply:
Done 👍🏼